content/overview/overview/why-available-other-agencies.md

```diff
@@ -5,22 +5,18 @@ menu:
 title: Why it’s available
 weight: 30
 aliases:
-  - /docs/intro/overview/why-available-other-agencies
-  - /intro/overview/why-available-other-agencies
+  - /overview/overview/why-available-other-agencies
+  - /overview/why-available-other-agencies
 ---
 
-Just as it has for us at 18F, we think cloud.gov will help other federal agencies deliver taxpayer-funded services the American people deserve in a faster, more user-centered way.
+cloud.gov will help other federal agencies deliver taxpayer-funded services the American people deserve in a faster, more user-centered way.
```

Comment:
Suggest changing "cloud.gov will help other federal agencies" to "cloud.gov helps federal agencies".